pkg/xds/envoy/listeners/kafka_configurer.go

```diff
@@ -0,0 +1,41 @@
+package listeners
+
+import (
+	envoy_listener "github.com/envoyproxy/go-control-plane/envoy/api/v2/listener"
+	envoy_kafka "github.com/envoyproxy/go-control-plane/envoy/config/filter/network/kafka_broker/v2alpha1"
+
+	"github.com/kumahq/kuma/pkg/util/proto"
+)
+
+const (
+	kafkaStatPrefix = "kstat"
+)
+
+func Kafka() FilterChainBuilderOpt {
+	return FilterChainBuilderOptFunc(func(config *FilterChainBuilderConfig) {
+		config.Add(&KafkaConfigurer{})
+	})
+}
+
+type KafkaConfigurer struct {
+}
+
+func (c *KafkaConfigurer) Configure(filterChain *envoy_listener.FilterChain) error {
+	pbst, err := proto.MarshalAnyDeterministic(
+		&envoy_kafka.KafkaBroker{
+			StatPrefix: kafkaStatPrefix,
```

Comment:
This should be listener name, look at TCP Proxy stat prefix and pass the same value